gr-azure-software-radio/README.md

```diff
@@ -43,45 +43,55 @@ NOTE: If you have installed the Azure CLI with the default apt package on Ubuntu
     sudo make install
     ```
 
-### Running the Tests
-To run the QA tests for the blob blocks, you must first create a storage account on Azure and 
-obtain the connection string. Next, export this connection string into an environment variable named
-"AZURE_STORAGE_CONNECTION_STRING". 
+### Running the Unit Tests
+Run the QA tests with any of the following methods:
+ - From the terminal you'll use to run the tests, run:
+    
+    From the build directory:
+    ```
+    make test
+    ```
+
+ - Or from the python directory:
+    ```
+    python -m pytest qa_*
+    ```    
+
+### Running the Integration Tests
+To run the integration tests for the blob blocks, you must first create a storage account on Azure 
+and obtain the connection string. Next, export this connection string into an environment variable 
+named "AZURE_STORAGE_CONNECTION_STRING". 
 
 Also generate a SAS token for the storage account with at least read and list permissions so we can
 test out all the auth options for the blob blobs. 
 
-The blob QA code requires an environment variable named "AZURE_STORAGE_URL" containing the URL to 
-the storage account, trailing '/' is optional. It also requires a AZURE_STORAGE_SAS environment
-variable containing just the SAS token string for the blob storage account to test against. 
+The blob integration test code requires an environment variable named "AZURE_STORAGE_URL" containing 
+the URL to the storage account, trailing '/' is optional. It also requires a AZURE_STORAGE_SAS
+environment variable containing just the SAS token string for the blob storage account to test
+against. 
 
 Finally, you must have at least one set of credentials supported by DefaultAzureCredential in your
 environment that has permissions to the blob account to test against. Running `az login` should be
 sufficient to provide this. 
 
-The QA code will create a randomly generated container to store
+The integration test code will create a randomly generated container to store
 unit test data requiring interactions with actual Azure infrastructure. 
 
 Tests can be run with any of the following methods:
  - From the terminal you'll use to run the tests, run:
-   ```
-   az login
-   ```
-
- - Then, from the build directory:
     ```
-    make test
+    az login
     ```
 
- - Or from the python directory:
+ - Then, from the python directory:
     ```
-    python -m pytest qa_*
+    python -m pytest integration_*
     ```
     
     or
 
     ```
-    python3 -m unittest qa_*
+    python3 -m unittest integration_*
```

Comment:
Anyway to run with ctest? Just curios .

Reply:
I specifically went against having it work with ctest. I don't think we want the integration tests running for 'make test' and I didn't have a clean way to have ctest run certain tests conditionally. 

If there's value in getting it working we can put something in the backlog.